src/libraries/System.Private.CoreLib/src/System/Exception.cs

```diff
@@ -86,7 +87,9 @@ public virtual Exception GetBaseException()
 
         public virtual string? Source
         {
-            get => _source ??= CreateSourceName();
+            [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
+                Justification = "The API will return <unknown> if the metadata for current method cannot be established.")]
+            get => _source ??= HasBeenThrown ? (TargetSite?.Module.Assembly.GetName().Name ?? "<unknown>") : null;
```

Comment:
Should `<unknown>` be localized?

Reply:
We do not localize strings like this. For example, `Module.FullyQualifiedName` return non-localized `<Unknown>`.

(This behavior was these even before my change. I have just unified it across runtimes. It is only applicable for NativeAOT with metadata stripped and maybe other corner-case situations.)